@shared/utils/formatContractError.ts

```diff
@@ -35,6 +35,8 @@ export function formatContractError(error: TransactionError | Error) {
   }
 
   switch (error?.message) {
+    case 'Connector not found':
+      return 'You must connect your wallet to place a bid'
```

Comment:
Please forgive me with the drive-by suggestion! 🚗 

To stay consistent with its sibling message, it would be nice to end this sentence with punctuation:

```suggestion
      return 'You must connect your wallet to place a bid.'
```